src/python/pants/VERSION

```diff
@@ -1 +1 @@
-1.6.0rc0
+1.6.0rc1
```

Comment:
Subsequent version bump for stable branch does not go on master but directly into the branch. E.g. https://github.com/pantsbuild/pants/pull/5643/files

Reply:
this PR is against the `1.6.x` branch - tho I guess it should have been a release-note only change against `master` and cherrypicked into `1.6.x` + a manual version bump?

will invert that when landing for this PR.

Reply:
Yeah the goal is to land this note on master, because doc site is published from master.

Then in 1.6.x branch, you can cherry pick this change in. then manually bump the version.

Reply:
hm, ok let me redo this then. thanks.